src/Tizen.NUI/src/internal/Interop/Interop.Window.cs

```diff
@@ -274,6 +274,18 @@ internal static partial class Window
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Window_ExcludeInputRegion")]
             public static extern void ExcludeInputRegion(global::System.Runtime.InteropServices.HandleRef window, global::System.Runtime.InteropServices.HandleRef inputRegion);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Window_Maximize")]
+            public static extern void Maximize(global::System.Runtime.InteropServices.HandleRef window, bool maximize);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Window_IsMaximized")]
+            public static extern bool IsMaximized(global::System.Runtime.InteropServices.HandleRef window);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Window_Minimize")]
+            public static extern void Minimize(global::System.Runtime.InteropServices.HandleRef window, bool minimize);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Window_IsMinimized")]
+            public static extern bool IsMinimized(global::System.Runtime.InteropServices.HandleRef window);
```

Comment:
아래 나온 src/Tizen.NUI/src/public/Window/Window.cs#L191, src/Tizen.NUI/src/public/Window/Window.cs#L241 의 warning 에 대해서 말씀드립니다.

비록 수정하신 부분이 아니더라도, cs file 에 걸쳐있는 warning 은 원칙적으로 새로 들어오는 패치에 대해서는 가능한 모든 warning 을 제거 하기로 했습니다.

아래 예시 확인 부탁드립니다.

        //  this is already deprecated, so suppress warning here.
        [System.Diagnostics.CodeAnalysis.SuppressMessage("Naming", "CA1717:Only FlagsAttribute enums should have plural names", Justification = "<Pending>")]

=> deprecated 되어 있으므로 이런식으로 추가하시면 될 것 같습니다.


src/Tizen.NUI/src/public/Window/Window.cs#L1365, src/Tizen.NUI/src/public/Window/Window.cs#L1463 는 `val.Dispose();` 를 `return;` 직전에 호출해 주시면 될 것 같습니다.

Reply:
네, 예시대로 수정하였습니다.

Reply:
1) src/Tizen.NUI/src/public/Window/Window.cs#L191 는 

        [Obsolete("Please do not use! This will be removed. Please use Window.EffectState instead!")]
        [EditorBrowsable(EditorBrowsableState.Never)]

기존것은 유지하고 아래 처럼 추가해 주셨으면 합니다. (즉 `[Obsolete(....)]` 가 유지되어야 합니다.)

        [Obsolete("Please do not use! This will be removed. Please use Window.EffectState instead!")]
        [EditorBrowsable(EditorBrowsableState.Never)]
        //  this is already deprecated, so suppress warning here.
        [System.Diagnostics.CodeAnalysis.SuppressMessage("Naming", "CA1717:Only FlagsAttribute enums should have plural names", Justification = "<Pending>")]

2) src/Tizen.NUI/src/public/Window/Window.cs#L241 도
기존것은 유지하고 추가해 주셨으면 합니다.

        [Obsolete("Please do not use! This will be removed. Please use Window.EffectType instead!")]
        [EditorBrowsable(EditorBrowsableState.Never)]
        //  this is already deprecated, so suppress warning here.
        [System.Diagnostics.CodeAnalysis.SuppressMessage("Naming", "CA1717:Only FlagsAttribute enums should have plural names", Justification = "<Pending>")]

3) src/Tizen.NUI/src/public/Window/Window.cs#L1365 도
`val.Dispose();` 를 `return;` 전에 추가해 주셨으면 합니다.

Reply:
네, 말씀하신 대로 수정하였습니다.